internal/engine/upper.go

```diff
@@ -131,40 +133,48 @@ func (u Upper) CreateManifests(ctx context.Context, manifests []model.Manifest,
 			case <-ctx.Done():
 				return ctx.Err()
 			case event := <-sw.events:
-				oldState := buildStates[event.manifest.Name]
-				buildState := oldState.NewStateWithFilesChanged(event.files)
-				buildStates[event.manifest.Name] = buildState
-
-				spurious, err := buildState.OnlySpuriousChanges()
-				if err != nil {
-					logger.Get(ctx).Infof("build watch error: %v", err)
-				}
+				if eventContainsConfigFiles(event) {
```

Comment:
question: when you edit the Tiltfile manually with Servantes, what happens? does it do reload the Tiltfile 7 times, one for each service?

Reply:
(this might be an ok failure case for now, i'm just curious what happens because i'm honestly not sure)

Reply:
Yes it redeploys all of the services.